jobs/build/custom/Jenkinsfile

```diff
@@ -191,6 +192,20 @@ node {
                             buildlib.buildBuildingPlashet(version, release, 7, true, auto_signing_advisory)  // build el7 embargoed plashet
                             buildlib.buildBuildingPlashet(version, release, 8, false, auto_signing_advisory)  // build el8 unembargoed plashet
                             buildlib.buildBuildingPlashet(version, release, 7, false, auto_signing_advisory)  // build el7 unembargoed plashet
+
+                            if(params.COMPOSE)
+                                notificationMessage = """
+                                    *:alert: custom ocp4 build compose ran during automation freeze*
+                                    COMPOSE parameter was set to true that forced build compose during
+                                    automation freeze."""
```

Comment:
i think these might get indented this way in slack which might look funny... not really a concern but we should probably try it out to see